docs/creating/scenarios.md

```diff
@@ -37,33 +37,6 @@ Scenarios consist of a name and a list of override names which together form tha
 Scenarios and overrides can be used to generate scripts that run a single Calliope model many times, either sequentially, or in parallel on a high-performance cluster
 (see the section on [generating scripts to repeatedly run variations of a model](../advanced/scripts.md)).
 
-## Importing other YAML files in overrides
-
-When using overrides, it is possible to have [`import` statements](yaml.md#relative-file-imports) for more flexibility.
-This can be useful if many overrides are defined which share large parts of model configuration, such as different levels of interconnection between model zones
-The following example illustrates this:
-
-```yaml
-overrides:
-  some_override:
-    techs:
-      some_tech.constraints.flow_cap_max: 10
-    import: [additional_definitions.yaml]
-```
-
-`additional_definitions.yaml`:
-
-```yaml
-techs:
-  some_other_tech.constraints.flow_out_eff: 0.1
-```
-
-This is equivalent to the following override:
-
-```yaml
-overrides:
-  some_override:
```

Comment:
@brynpickering I noticed this is no longer true after the most recent changes to YAML #719. If needed, this could be brought back in a separate issue. Let me know your preference.

Reply:
No, I certainly never used this in practice so it was likely adding more complexity than necessary.